styleguide.css

```diff
@@ -135,6 +139,15 @@ summary {
     color: #00283C;
 }
 
+.icon-logo-cafebabel .path1:before {
```

Comment:
What are these for?

Reply:
told you it was the way for icomoon to have 1 icon with severals colors

Reply:
Didn't you end up making the logo in SVG so you have a proper logo and you don't need to tweak this?

Reply:
yep

Reply:
Ok, that's why I was surprise to see these lines coming up again and was confused.
This PR being closed I guess we need to clean this in a new PR ;)

Reply:
ok